### PR TITLE
[BACKPLANE-1236] tidy up 

### PR DESCRIPTION
**Pull request by neeasade**
- add linting
- conform test namespace layout
- add CODEOWNERS
- kaocha

### CODE REVIEW

**Review by liamchzh — APPROVED** [review]

File: project.clj [hunk 1]
```diff
@@ -2,9 +2,31 @@
...
+
   :dependencies []
-  :plugins [[lein-cloverage "1.2.2"]]
-  :profiles {:dev {:dependencies [[org.clojure/clojure "1.10.2"]]}}
+
+  :plugins [[lein-cloverage "1.2.2"]
+            [jonase/eastwood "0.3.14" :exclusions [org.clojure/clojure]]]
+
+  :eastwood {:exclude-linters
+             [ ;; clj-kondo will catch the wrong-arity cases
```
Comment (calvis) [inline project.clj:12]:
```suggestion
             [;; clj-kondo will catch the wrong-arity cases
```
shakes fist at emacs autoindent

**Review by calvis — APPROVED** [review]

File: project.clj (hunk 1 above)
Comment (neeasade) [reply project.clj]: annoyed at this one b/c it had me sus/questioning the right indent

File: .circleci/config.yml [hunk 2]
```diff
@@ -13,16 +23,17 @@ jobs:
...
       - run:
           name: Code Coverage
-          command: lein trampoline cloverage --codecov
-      - run:
-          name: Upload Coverage
-          command: bash <(curl -s https://codecov.io/bash) -f target/coverage/codecov.json
-          when: always
+          command: lein cloverage --codecov
+      - codecov/upload:
+          file: target/coverage/codecov.json
```
Comment (calvis) [inline .circleci/config.yml:36]: @neeasade this comment got resolved but not addressed?

Comment (neeasade) [reply .circleci/config.yml:36]: my bad (slow down) --> https://github.com/circleci/bond/pull/57

**Pull request merged**